app/components/Admin/AddReviewCommentModal.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {useState} from 'react';
+import {Modal, Button, Form, FormCheck} from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faLock, faEye} from '@fortawesome/free-solid-svg-icons';
+
+interface Props {
+  show: boolean;
+  title: string;
+  onSubmit: (detail: object) => void;
```

Comment:
```suggestion
  onSubmit: (detail: {commentText: string, isInternalComment: boolean}) => void;
```

The `object` primitive type is only useful the `Object` API, otherwise not so much, as it doesn't allow you to access properties of the object